packages/systemd/systemd-sysusers.conf

```diff
@@ -0,0 +1,5 @@
+[Unit]
+After=selinux-policy-files.service
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
Do we need to include this? It's not part of the default unit. `local-fs.target` is also pretty late considering that we'll have started all the services that need users created before then.

Reply:
Maybe not? I was confused by when this unit starts if it doesn't have an `Install` section, but probably it isn't needed. I'll test and confirm.

Reply:
I confirmed the `Install` section isn't needed, I removed it and I still saw `systemd-sysusers` running after the SELinux policy files were copied.